docs/src/bpmn-workflows/event-subprocesses/event-subprocesses.md

```diff
@@ -0,0 +1,103 @@
+# Event Subprocess
+
+An event subprocess is a subprocess that can be added to any process or subprocess and can be triggered by either a timer or a message start event.
+
+![event-subprocess](/bpmn-workflows/event-subprocesses/event-subprocess.png)
+
+A start event's subscription or timer is created when the scope containing the event subprocess is activated and removed when the same scope is terminated or completed. Event subprocesses can be thought of as (sub)process level boundary events that are inside of the (sub)process scope instead of being attached to it. Like boundary events, they can be configured as *interrupting* or *non-interrupting*. In the case of *interrupting* event subprocesses, when the start event occurs, all active execution paths (including other non-interrupting event subprocesses) are terminated. After the other activated execution paths terminate, the triggered event subprocess is activated. 
+
+## Variables
+
+Even though, at first glance, event subprocesses seem to have similar behaviors to boundary events, it's important to note that boundary events reside *outside* of the scope of the element that they are attached to. This means that they are not aware of the variables pertaining to that element. The ability to access these variables is crucial in compensation use cases and is the motivation for the event subprocess. Since the event subprocess resides inside the scope of the (sub)process, it can access its variables.
```

Comment:
```suggestion
Unlike a boundary event, an event subprocess is inside the scope. So, it can access and modify **all local variables** of its containing scope. This is not possible with a boundary event because a boundary event is outside of the scope.
```